decidim-admin/app/controllers/decidim/admin/application_controller.rb

```diff
@@ -4,7 +4,14 @@ module Admin
     # The main application controller that inherits from Rails.
     class ApplicationController < ActionController::Base
       include NeedsOrganization
+      include Pundit
       protect_from_forgery with: :exception, prepend: true
+
+      after_action :verify_authorized
```

Comment:
We should have a default rescue when Pundit fails:

https://github.com/elabs/pundit#rescuing-a-denied-authorization-in-rails